packages/dso-toolkit/src/styles/dso.scss

```diff
@@ -12,7 +12,7 @@
 // @import "components/attachments-counter"; -> Moved to @dso-toolkit/styling
 // @import "components/alert"; -> Moved to @dso-toolkit/styling
 // @import "components/anchor"; -> Moved to @dso-toolkit/source/src/styling/global/a
-@import "components/application-heading";
+// @import "components/application-heading"; -> Moved to @dso-toolkit/styling
```

Comment:
Eigenlijk niet. Voorheen administreerden we dit wel zo, maar toen bestond inderdaad `@dso-toolkit/styling` nog. Nu is het wellicht beter om het echte verhuisdoel aan te duiden.